.github/workflows/clang-format-checker.yml

```diff
@@ -26,9 +26,10 @@ jobs:
           fetch_depth: 100 # Fetches only the last 10 commits
 
       - name: "Listed files"
+        env: LISTED_FILES: ${{ steps.changed-files.outputs.all_changed_files }}
         run: |
           echo "Formatting files:"
-          echo "${{ steps.changed-files.outputs.all_changed_files }}"
+          echo $LISTED_FILES
```

Comment:
Should there be quotes around `$LISTED_FILES` here?

Reply:
Yup!